composer/datasets/cifar.py

```diff
@@ -65,3 +66,48 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataloaderHpara
                                                     batch_size=batch_size,
                                                     sampler=sampler,
                                                     drop_last=self.drop_last)
+
+
+@dataclass
+class CIFAR10WebDatasetHparams(JpgClsWebDatasetHparams):
+    """Defines an instance of the CIFAR-10 WebDataset for image classification."""
+
+    dataset_s3_bucket = 'mosaicml-internal-dataset-cifar10'
```

Comment:
I think these MosaicML-specific bucket paths should probably be removed, or replaced with a more abstract 'your-s3-bucket' string.

Reply:
Done